how-to/about-how-to.md

```diff
@@ -47,7 +47,12 @@ Make sure you meet the following prerequisites before following the steps:
 
 ### About the "Step by step guide" sections
 
-..
+When you are explaining steps in a process, it can often be useful to include a screenshot for each part of the process. One of the many reasons for having a screenshot for every key step is that it helps someone who is evaluating the software to see how the software will work, even without having to install the software.
+
+In the template-howto.adoc file, you'll notice a table in the "Step-by-step" section of the template. The table allows you easily add a screenshot or callout next to each step.
```

Comment:
"The table allows you easily add a screenshot or callout next to each step." should be changed to "The table allows you to easily add a screenshot or callout next to each step."

Reply:
@Dishebh would you be able to make the fix that I indicated in this comment? The word "to" is missing.